cmd/docker-notation/pull.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/distribution/distribution/v3/reference"
+	"github.com/notaryproject/notation-go-lib"
+	"github.com/notaryproject/notation/cmd/docker-notation/docker"
+	ios "github.com/notaryproject/notation/internal/os"
+	"github.com/notaryproject/notation/pkg/config"
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/urfave/cli/v2"
+)
+
+var pullCommand = &cli.Command{
+	Name:      "pull",
+	Usage:     "Pull an image or a repository from a registry",
```

Comment:
not sure if `or a repository` is necessary

Reply:
It is from
```
$ docker pull
"docker pull" requires exactly 1 argument.
See 'docker pull --help'.

Usage:  docker pull [OPTIONS] NAME[:TAG|@DIGEST]

Pull an image or a repository from a registry
```

Reply:
It is not meaningful to pull all tags for `docker notation pull`. Will update accordingly.